app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Admin::DashboardController < ApplicationController
   def index
+    @uncontacted_companies = Company.where(status: "uncontacted")
+    @contacted_companies = Company.where(status: "contacted")
```

Comment:
We're thinking about moving these into a presenter class since we'll have a few more.  We'll do that in another branch since Bhargavi needed access to what we did so far for her to get started on the admin/companies_controller